docs/detections/alerts-visualizations.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[visualize-alerts]]
+== Visualize detection alerts
+
+Visualize and group detection alerts by specific parameters in the visualization section of the Alerts page. 
+
+[role="screenshot"]
+image::images/alert-page-visualizations.png[]
+
+Use the left menu to select a view type (*Trend*, *Table*, or *Treemap*), and use the right menus to select the ECS fields to use for grouping: 
+
+* *Group by*: Primary field for grouping alerts.
+* *Group by top* (if available): Secondary field for further subdividing grouped alerts.
+
+For example, you can group first by rule name (`Group by: kibana.alert.rule.name`), then by host name (`Group by top: host.name`) to visualize which detection rules generated alerts, and which hosts triggered each of those rules. For groupings with a lot of unique values, the top 1,000 results are displayed.
+
+NOTE: The *Group by top* option isn't available in all view types. You can also leave *Group by top* blank to group by only the primary field in *Group by*.
+
+To reset a view to default settings, click the settings menu (image:images/three-dot-icon.png[Settings menu icon,18,18]) in the upper-right and select *Reset group by fields* (the settings menu appears when hovering over the area).
+
+=== Trend
+The trend view shows the occurrence of alerts over time. This is the default visualization on the Alerts page, and by default it groups alerts by detection rule name (`kibana.alert.rule.name`).
+
+NOTE: The *Group by top* menu is unavailable for the trend view.
+
+[role="screenshot"]
+image::images/alerts-viz-trend.png[Trend visualization for alerts]
+
+=== Table
+The table view shows the count of alerts in each group. By default, the table groups alerts first by detection rule name (`kibana.alert.rule.name`), then by host name (`host.name`).
```

Comment:
```suggestion
The table view shows the count of alerts in each group. By default, it groups alerts by detection rule name (`kibana.alert.rule.name`) first, then by host name (`host.name`).
```

Reply:
@nastasha-solomon I'm still leaning toward a more parallel form "first by x, then by y." I also think that "first" kind of gets lost after the long field name in parentheses.
```suggestion
The table view shows the count of alerts in each group. By default, it groups alerts first by detection rule name (`kibana.alert.rule.name`), then by host name (`host.name`).
```

Reply:
No strong feelings either way :)